test/Table_Tests/src/Common_Table_Operations/Cross_Tab_Spec.enso

```diff
@@ -130,3 +132,14 @@ spec setup =
         Test.specify "should not allow Group_By for values" <|
             err1 = table.cross_tab [] "Key" values=[Count, Group_By "Value"] on_problems=Problem_Behavior.Ignore
             err1.should_fail_with Illegal_Argument.Error
+
+        Test.specify "should fail on invalid aggregations" <|
+            table = table_builder [["Key", ["x", "x", "x", "x", "y", "y", "y", "z", "z"]], ["TextValue", ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i']]]
+            [Problem_Behavior.Report_Error, Problem_Behavior.Report_Warning, Problem_Behavior.Ignore].each pb-> Test.with_clue "Problem_Behavior="+pb.to_text+" " <|
+                err = table.cross_tab [] "Key" values=[Average "TextValue"] on_problems=pb
+                case setup.is_database of
+                    True ->
+                        err.should_fail_with SQL_Error
+                    False ->
+                        err.should_fail_with Invalid_Aggregation
+                        err.catch . should_equal (Invalid_Aggregation.Error "" [0, 4, 7] "Cannot convert to a number.")
```

Comment:
Currently `Invalid_Aggregation` is only reported for in-memory, for Database we will be getting a SQL error.

I assume the specification of this task is only to turn the existing `Invalid_Aggregation` warnings into errors, not also align the SQL errors to turn them into `Invalid_Aggregation`. Is that correct @jdunkerley ?